### PR TITLE
Action bar gets cut off resolved #520

### LINKED ISSUE #520: Action bar gets cut off on API 16 emulator
**Is it a ?**

- [x] Bug Report

### Step 1: Are you ready to report?
   - Your issue may already be reported! Please search on the [issue track](../) before creating one.
   - [x] I have tested with the last (dev) version of the software.  ( If not, and if possible, run your test again with the most recent version available.)

### Step 2: What is your environment?
  * version of the software you use : 2.3 build 55
  * device / operating system : Android Studio API 16 Emulator and Genymotion API 16 Emulator
  
### Step 3: Describe the problem:

  * What do you obtain? A copy of the error message or a screenshot is always useful.

    The `ActionBar` gets cut off while running on API 16 emulator.

    **Screenshot from Android Studio API 16 Emulator**
   
screenshot_1519581206

   **Screenshot from Genymotion API 16 Emulator**

api16_1

  * When does this occur?

    Opening the app. 

#### Steps to reproduce:

  1. Open the app
  
  * What should be the behaviour from your point of view? How do you expect the service to work?

    `ActionBar` must not be cut off.

**I want to verify if anyone else is facing this issue on any other API level or is it just me?**

### PR DESCRIPTION
**Pull request by vansh1sh**
Fixes #520 

Changes: Added the fix for lower api versions, it removes the white space/cut off issue between the action bar and the status bar in lower API levels.

Screenshots/GIF for the change: N/A

@mhutti1 @brijeshshah13 Please check.

### CODE REVIEW

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 1]
```diff
@@ -377,6 +377,11 @@ public void onCreate(Bundle savedInstanceState) {
 
     initPlayStoreUri();
 
+    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP){
+    } else{
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Formatting 

if () {

} else {

}

use static imports for the SDK_INT stuff if possible.

Comment (vansh1sh) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: @mhutti1 Should this be fine?

import static android.os.Build.VERSION.SDK_INT;
import static android.os.Build.VERSION_CODES;
 . 
 .
 .
 if (SDK_INT > VERSION_CODES.LOLLIPOP){
      
    } else{
      snackbarLayout.setFitsSystemWindows(true);
    }

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 2]
```diff
@@ -377,6 +379,12 @@ public void onCreate(Bundle savedInstanceState) {
 
     initPlayStoreUri();
 
+    if (SDK_INT > VERSION_CODES.LOLLIPOP){
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Please use if ! if you are just going to use one branch. Spacing is still way off from style guidelines.

Comment (vansh1sh) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: @mhutti1 Fixed it. Does it look correct now?

**Pull request merged**